docs/admin/troubleshooting_updates.rst

```diff
@@ -104,8 +104,50 @@ Note that ``dom0`` and ``apply_dom0`` are separate steps.
    for assistance.
 
 4. Reboot the system. ``dom0`` updates are often
-   security-sensitive, and may requrie a reboot to take
+   security-sensitive, and may require a reboot to take
    effect.
+   
+GPG check failures
+^^^^^^^^^^^^^^^^^^
+
+If at any point you see a message that the GPG check failed,
+and you see an indication that a key has expired, your system
+may be trying to use an old copy of the SecureDrop Release Signing
+Key. 
+
+To remove the expired key and replace it with the new, valid key, you
+can perform the following steps:
+
```

Comment:
(see above, and I would remove "replace it with the new valid key" in favour of wording above or similar)